frontend/config-assistant/src/components/gui-editor/PropertyMetadata.vue

```diff
@@ -64,6 +76,12 @@ function getTypeDescription(): string {
   if (props.nodeData.schema.enum) {
     return 'enum';
   }
+  if (props.nodeData.schema.oneOf.length > 0) {
```

Comment:
I don't like that too much. AnyOf or OneOf isn't really a type

Reply:
this is something to discuss. It is not a type, but it has an explanatory effect and helps the user understand the dropdown menu. We need at least some way to show what the dropdown menu is about (or the multiselect for anyOf)